examples/Cpp/MacOS/HighResolution.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
See comment above: is this `.plist`file backwards compatible for older Macs without a retina display?